pkg/ebpfeng/ebpf_engine_interface.go

```diff
@@ -0,0 +1,7 @@
+package ebpfeng
+
+type EbpfEngineClient interface {
+	StartEbpfEngine() error
```

Comment:
[TODO]: I don't see any need in repeating the fact that this is `EbpfEngine`. How about `GetData` instead?

Reply:
done